package.json

```diff
@@ -123,7 +123,9 @@
     "not ie < 11",
     "not op_mini all"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "storybook": "6.5.13"
```

Comment:
There should only be devDependencies in root `package.json`.

This was missing before, did it work because some other dependency installed it?